spec/support/test_server.rb

```diff
@@ -17,7 +17,7 @@
       protocol: 'pbc',
       nodes: [
         {
-          host: "docker",
+          host: "localhost",
```

Comment:
before I changed this, I was getting the error: `getaddrinfo: nodename nor servname provided, or not known` - I'm guessing this has something to do with my docker/local setup, so happy to change back